src/decorator/typechecker/IsBigInt.ts

```diff
@@ -0,0 +1,27 @@
+import { ValidationOptions } from '../ValidationOptions';
+import { buildMessage, ValidateBy } from '../common/ValidateBy';
+
+export const IS_BIGINT = 'isBigInt';
+
+/**
+ * Checks if the value is a bigint
+ */
+export function isBigInt(val: unknown): val is bigint {
+  return typeof val === 'bigint';
+}
+
+/**
+ * Checks if the value is a bigint
+ */
+export function IsBigInt(validationOptions?: ValidationOptions): PropertyDecorator {
+  return ValidateBy(
+    {
+      name: IS_BIGINT,
+      validator: {
+        validate: (value, args): boolean => isBigInt(value),
+        defaultMessage: buildMessage(eachPrefix => eachPrefix + '$property must be a bigint number', validationOptions),
```

Comment:
Should this say `BigInt` instead of `bigint`?

Reply:
I think not. It should say `bigint` in order to stay consistent with the error message for `IsNumber` validator from the same folder.

```
defaultMessage: buildMessage(
  eachPrefix => eachPrefix + '$property must be a number conforming to the specified constraints',
  validationOptions
),
```

https://github.com/typestack/class-validator/blob/63fe9c532fb6f633ddbb90e8225c5af49750513a/src/decorator/typechecker/IsNumber.ts#L54C9-L57C11